### ISSUE TITLE
Can't read any TS (local or from server) if api for signature verification is down

### ISSUE THREAD

**Issue by James-Sangalli** [AlphaWallet/alpha-wallet-android#1422]

**Comment by JamesSmartCell** [comment]
@ChintanRathod can you have a look at this one. The code in question is in class AlphaWalletService here:

public XMLDsigDescriptor checkTokenScriptSignature(File tokenScriptFile)

it's used to check the signature of the tokenscript. Apparently if the server is down the app is prevented from downloading the script.

Recommend you delete scripts (in the data/data/io.stormbird.wallet/files directory, all the .xml files) then watch this wallet:

0xD40EBAaA4Cb7fC3a5D7B373309F68B2ABCe5F1C5

then open up the GFO (GFO) token with 11,690 balance (or thereabouts). The tokenscript should start downloading here:

```public void checkForNewScript(Token token)```

In class TokenFunctionViewModel

**Comment by ChintanRathod** [comment]
I am not able to reproduce this issue. I did follow steps, 
- Watch 0xD40EBAaA4Cb7fC3a5D7B373309F68B2ABCe5F1C5 account
- Tap on GFO with value 11,690
- Check for data/data/io.stormbird.wallet/files which is blank after tapping 
- Also checked in TokenScript Management where no files with 0xc132bd...

![image](https://user-images.githubusercontent.com/4371780/85704519-fdd81780-b6fd-11ea-892d-505cbbfb7d14.png)

![image](https://user-images.githubusercontent.com/4371780/85704682-23fdb780-b6fe-11ea-91fb-4c98c12557a3.png)

Did I miss anything to reproduce? Could you please check.

**Comment by JamesSmartCell** [comment]
Yes, you need to fake that the API is down - change the API calling code so it returns an error, then investigate why the API can't complete if there is an error at that stage.
